### PR TITLE
Bump Resolwe dependencies

### PR DESCRIPTION
**Pull request by gregorjerse**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -65,8 +65,8 @@
         "redis~=3.5.3",
         "resolwe-runtime-utils~=3.0.0",
         "shellescape~=3.8.1",
-        "Sphinx>=1.5.1, <1.7.0",
         "beautifulsoup4~=4.9.3"
+        "Sphinx",
```
Comment (dblenkus) [inline setup.py:67]: Limit the version of the Sphinx.

**Review by dblenkus — COMMENTED** [review]
Looks good.

File: setup.py (hunk 1 above)
Comment (gregorjerse) [reply setup.py:67]: OK, will pin it to the current stable version.

File: setup.py [hunk 2]
```diff
@@ -41,11 +41,11 @@
     install_requires=[
         # XXX: Temporarily pin asgiref to 3.2.x since testing framework freezes
         # with version 3.3.x
-        "asgiref~=3.2.10",
+        "asgiref~=3.3.1",
```
Comment (dblenkus) [inline setup.py:42]: Remove the comment above if it was solved.

**Review by dblenkus — APPROVED** [review]

**Pull request merged**